_data/apps.yml

```diff
@@ -1081,3 +1081,13 @@
     - "gallery"
   license: "MIT"
   icon: "lightgallery.png"
+-
+  name: "Min"
+  description: "A smarter web browser"
+  website: "https://minbrowser.github.io/min/"
+  repository: "https://github.com/minbrowser/min"
+  keywords:
+    - "browser"
+    - "privacy"
+  license: "Apache 2.0"
+  icon: "icon256.png"
```

Comment:
@kevinsawicki How do I do that? https://github.com/electron/electron.atom.io/blob/gh-pages/CONTRIBUTING.md#adding-an-app-or-project-to-the-site doesn't mention it. Should I add it to https://github.com/electron/electron.atom.io/tree/gh-pages/images/apps ?